JustBrewIt/Pages/Brewery.cshtml.cs

```diff
@@ -13,14 +13,22 @@ public class BreweryModel : PageModel
     {
         public object MyBreweryAPI { get; set; }
 
+        public string Error;
```

Comment:
Initialize Error Variable.

Reply:
This is only used in the catch block, so keep it local to there.  No need to give it wider scope.